src/scheduler/fifo.c

```diff
@@ -1561,34 +1561,9 @@ run_update_resresv(status *policy, int pbs_sd, server_info *sinfo,
 			update_job_comment(pbs_sd, array, buf);
```

Comment:
So in the era of synchronous run_jobs if an array subjob use to fail because of MOM rejects, what use to be the comment set on the array parent?
I think it either stays as "Job Array has Begun" on array parent if the array parent in in 'B' state OR it changes to the job comment being set by scheduler if array parent is not in 'B' state.
If what I mentioned is true (and you may want to test it out :) ) then with Asynchronous run jobs the Array parent will always show "Job Array has Begun" comment even when it's very first subjob fails to run. So, in because of this shifting the comment to server makes sense. Also, moving the job comment for normal jobs to server and keeping for array jobs to scheduler looks inconsistent.